sites/automationworld/redirect-handler.js

```diff
@@ -0,0 +1,15 @@
+const redirect = (from, pattern) => pattern.exec(from);
+
+module.exports = ({ from }) => {
+  const digital = redirect(from, /\/sites\/default\/files\/digital_edition\/(.*)/);
+  if (digital && digital[1]) {
+    return { to: `https://digitaleditions.automationworld.com/${digital[1]}` };
```

Comment:
Move you're if and return to one line in all cases where max line length is not reached... e.g. 
```js
if (digital && digital[1]) return { to: `https://digitaleditions.automationworld.com/${digital[1]}` };
```

Do this in all handlers...

Reply:
I changed them all to exec, the shorter name made them all fit by 1 letter :P